### ISSUE TITLE
A11y_.NET Core_WinForms_ToolStripContainer_Bottomtool bar_Usability: When we navigate to "ToolStripContainer" dialog we are unable to activate the controls present in the bottom toolbar with Keyboard.

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9194]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev4;#DesktopApp;#Win11;#NonBenchmark;#A11yUsable

### Actual:
When we navigate to "ToolStripContainer" dialog we are unable to activate the controls present in the bottom toolbar.

### Expected:
When we navigate to "ToolStripContainer" dialog we must be able to activate the controls present in the bottom toolbar.

### User Impact:
Keyboard users will face difficulty to activate the controls if the controls are not accessible with Keyboard.
 
### Attachments:
![Bottom toolbar](https://github.com/dotnet/winforms/assets/125462007/6b5efda5-bf19-452d-bf53-c9ad6b930a4c)

### Steps to reproduce

1. Install and Open Winforms App.
2. "Main Form" screen would appear.
3. TAB to "MenuToolbars_Controls:Testing" the controls under "Menu Toolbars" Tab.
4. TAB to "ToolStripContainer" button.
5. Now navigate to "Bottom tool bar" section.
6. Now observe that when we navigate to the "Bottom tool bar" section we are unable to activate the controls present in the tool bar.

**Comment by 12-shweta** [comment]
#BM_.NET Core[WPF]_Win32_May23;

**Comment by Nora-Zhou01** [comment]
We add these buttons on from designer and set some properties for them, don't add any keyboard event, so there is no performing functionality when activated using keyboard.

**Comment by merriemcgaw** [comment]
Given this was marked as Sev4 we can close this.

**Comment by 12-shweta** [comment]
Github tags: #Closed;
